.travis.yml

```diff
@@ -225,3 +225,25 @@ jobs:
         RUN_FUNCTIONAL_TESTS=false
         GOAL="deploy"
         BITCOIN_CONFIG="--enable-gui --enable-reduce-exports --enable-werror"
+
+  - stage: test
+    name: 'macOS 10.10 native'
+    os: osx
+    # Use the most recent version:
+    # Xcode 11, macOS 10.14, JDK 12.0.1
+    # https://docs.travis-ci.com/user/reference/osx/#OS-X-Version
+    osx_image: xcode11
+    language:
+      - ruby
+      - python
+    python:
+        - "3.6.5"
```

Comment:
Should this be `3.5.6` to match our [minimum supported Python](https://github.com/bitcoin/bitcoin/blob/master/.python-version)?